server/controllers/userController.js

```diff
@@ -0,0 +1,39 @@
+import createToken from '../helpers/createToken'
+import bcrypt from 'bcrypt';
+import models from '../database/models';
+import responseMessage from '../helpers/responseMessage';
+
+/**
+ * @exports
+ * @class UserController
+ */
+const saltRounds = 10;
+const { User } = models;
+class UserController {
+  /**
+   * create a user
+   *
+   * @param {object} request The request.
+   * @param {object} response The response.
+   * @returns {object} response.
+   */
+  static async createUser(request, response) {
+    const { firstName, lastName, email, password, avatarUrl, verified } = request.body
+    try {
+      const existingUser = await User.findOne({ where: { email } })
+      if (existingUser) {
+        return responseMessage(response, 400, { message: 'User already exist' });
+      }
+      const newUser = await User.create({ firstName, lastName, email, password: bcrypt.hashSync(password, saltRounds), avatarUrl, verified })
+      if (newUser) {
+        const { id } = newUser.dataValues
+        const token = createToken({ id }, '24h');
+        return responseMessage(response, 201, { status: 'success', message: 'Sign up Successful', token });
+      }
+    } catch (error) {
+      return responseMessage(response, 500, { error: error.message });
```

Comment:
Use 'message' in the object, instead of 'error'. Don't forget to also add the 'status' property

Reply:
done